tapsend/send.go

```diff
@@ -172,7 +172,12 @@ type FundingDescriptor struct {
 // TapCommitmentKey is the key that maps to the root commitment for the asset
 // group specified by a recipient descriptor.
 func (r *FundingDescriptor) TapCommitmentKey() [32]byte {
-	return asset.TapCommitmentKey(r.ID, r.GroupKey)
+	assetSpecifier := asset.NewIdSpecifier(r.ID)
+	if r.GroupKey != nil {
+		assetSpecifier = asset.NewSpecifier(r.ID, *r.GroupKey)
+	}
+
```

Comment:
I think this can be simplified to: 
```
assetSpecifier := asset.NewSpecifierOptionalGroupKey(r.ID, r.GroupKey)
```

I think this was mentioned above, but perhaps there was some other resolution somewhere?

Reply:
Scratch that, I see later on we modified this to use the specifier.